drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -253,10 +259,20 @@ def external_form_submissions(**args)
       resolve_external_form_submissions(scope, **args)
     end
 
-    def ce_opportunities
+    def ce_opportunities(filters: nil)
+      raise unless Hmis::Ce.configuration.enabled?
+
+      scope = load_ar_association(object, :ce_opportunities)
+      scope = scope.where(status: filters&.status) if filters&.status.present?
+      scope.order(created_at: :desc)
```

Comment:
A little confusing to use the data loader since this field would have N+1 anyway from `where` and `order` clauses. Prefer removing data loader, so it's clearer to the reader that this field is N+1 and only should be resolved on a single Project.
```suggestion
      scope = object.ce_opportunities
      scope = scope.where(status: filters&.status) if filters&.status.present?
      scope.order(created_at: :desc)
```